Cart Flow/Custom Views/Cells/ItemListCell.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  ItemListCell.swift
+//  Cart Flow
+//
+//  Created by Scott Quintana on 8/5/20.
+//  Copyright © 2020 Scott Quintana. All rights reserved.
+//
+
+import UIKit
+
+protocol ItemListCellDelegate: class {
+    func didTapEditItemButton(for item: ShoppingItem)
```

Comment:
You seem to have a great grasp on the delegation pattern...well done! It took me forever to grok this pattern when I was first starting out, but you'll see it over and over again in iOS development.